presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -334,6 +334,7 @@ public String toCqlLiteral(Object prestoNativeValue)
             case ASCII:
             case TEXT:
             case VARCHAR:
+            case DATE:
```

Comment:
`prestoNativeValue` for date will be number of days since epoch 
in the `getColumnValueForCql(Row row, int position)` above we use yyyy-mm-dd representation

can we unify the two? if not, why would i `quoteStringLiteral` the number?